Framework/interface/RefVector.h

```diff
@@ -151,6 +163,21 @@ namespace panda {
     Indices* indices_{0};
   };
 
+  template<class E>
+  typename RefVector<E>::Indices
+  RefVector<E>::sort(ContainerBase::Comparison const& _c)
+  {
+    // define an index comparison using the element comparison
+    auto indexComp([this, &_c](short i1, short i2)->Bool_t {
+        return _c(static_cast<Element const&>((*container_)->elemAt(i1)),
+                  static_cast<Element const&>((*container_)->elemAt(i2)));
+      });
+
+    std::sort(indices_->begin(), indices_->end(), indexComp);
+
+    return *indices_;
```

Comment:
If we are just returning `*indices_`, it doesn't provide any additional information, so I think we should rather just return nothing (`indices_` is always accessible from the `indices()` function). By dereferencing and returning an rvalue, there is an implicit copy of index vector taking place.

Reply:
Ah sure. I had left the return value in to mirror `Collection::sort`, but I can drop it.